.ci/openshift-ci/images/entrypoint.sh

```diff
@@ -90,6 +90,31 @@ update_initrd_kernel_modules()
 	( cd "${rootfs_dir}" && find . | cpio -H newc -o | gzip -9 ) > ${initrd_path}
 }
 
+# Manage symlinks on host.
+handle_symlinks() {
+	local create_links_script="/tmp/create_links.sh"
+	cat <<-EOF > "${HOST_MOUNT}/${create_links_script}"
+	for t in /opt/kata/bin/*; do
+		executable="\$(basename \$t)"
+		executable_found="\$(which "\$executable" 2>/dev/null)"
+		if [[ -n "\$executable_found" && \
+			"\$(readlink "\$executable_found")" = "\$t" ]]; then
+			echo "Found \$executable_found"
+			continue
```

Comment:
Does "/var/usrlocal/bin/" have precedence over any other PATH? and if it does, would it make sense to just forcefully link all the executables regardless to whether executable_found or not?
Also do you recall what links were existed? do you know if this is expected?

Reply:
Hi @snir911 , yes,  `/var/usrlocal/bin/` is mounted in `/usr/local/bin` and it should have 1st precedence on PATH.

Reply:
@wainersm wouldn't `chroot "$HOST_MOUNT" sh -c 'for t in /opt/kata/bin/*; do ln --force -s "$t" /var/usrlocal/bin/; done'` be good enough?